src/Microsoft.ServiceHub.Framework/Extensions/ServiceHubUtilities.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.ServiceHub.Framework.Extensions
+{
+	/// <summary>
+	/// Contains utility methods to be used by ServiceHub.
+	/// </summary>
+	internal static class ServiceHubUtilities
```

Comment:
Can you please add a prominent comment on each internal type that must remain but isn't obviously useful to this library directly, so that I don't delete it later?

Reply:
Done.